### DISCUSSION TITLE
Noção básica de programação EM C#

### DISCUSSION THREAD

**Discussion by jiogo1** [frontendbr/forum#2475] [Perguntas e Respostas]
Aceito recomendações para materiais de estudo em C# e recomendações de outras linguagens que seriam interessantes saber o básico.

totalmente Iniciante na área

**Comment by ghost** [comment]


**Reply by jiogo1** [reply]
Opa obrigado, vou dar uma lida

**Comment by felquis** [comment]
Bot detectado
